tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -65,6 +68,28 @@ def testee(a: cases.IJKField) -> cases.IJKField:
     cases.verify_with_default_data(cartesian_case, testee, ref=lambda a: a)
 
 
+def test_gtfn_file_cache(cartesian_case):
```

Comment:
Can this test become a unittest? All tests in this file are run through all backends, while this is a special test for a specific backend.

Reply:
Yes, that's a good point. I placed it there because it tests the whole pipeline, but I am happy to move it to test_gtfn_module.py or any other place you suggest.

Reply:
I moved it to test_gtfn_module.py.